respy/pre_processing/model_processing.py

```diff
@@ -122,6 +122,9 @@ def parse_parameters(params, paras_type="optim"):
         optim_paras["type_shares"] = np.zeros(2)
         optim_paras["type_shifts"] = np.zeros((1, 4))
 
+    # ignore meas error for non wage sectors
```

Comment:
Can we remove this? We could infer the number of choices with wage and without from the index and then expand the measurement error with zeros. This would also be useful for the new state space which needs the information on choices with experiences and without.

Reply:
I agree with the first part, but we can't use this to infer the number of choices with experiences. For example, edu has experience but no wage.

Reply:
Done.